pkg/splunk/enterprise/configuration.go

```diff
@@ -949,11 +947,11 @@ func validateSplunkAppSources(appFramework *enterpriseApi.AppFrameworkSpec, loca
 
 	}
 
-	if localScope && appFramework.Defaults.Scope != "" && appFramework.Defaults.Scope != "local" {
+	if localScope && appFramework.Defaults.Scope != "" && appFramework.Defaults.Scope != enterpriseApi.ScopeLocal {
 		return fmt.Errorf("Invalid scope for defaults config. Only local scope is supported for this kind of CR")
 	}
 
-	if appFramework.Defaults.Scope != "" && appFramework.Defaults.Scope != "local" && appFramework.Defaults.Scope != "cluster" {
+	if appFramework.Defaults.Scope != "" && appFramework.Defaults.Scope != enterpriseApi.ScopeLocal && appFramework.Defaults.Scope != enterpriseApi.ScopeCluster {
```

Comment:
Should we also add `enterpriseApi.ScopeClusterWithPreConfig` in the if condition?